src/main/lua/exasolrls/RlsAdapter.lua

```diff
@@ -4,7 +4,7 @@ local QueryRewriter = require("exasolrls.QueryRewriter")
 
 -- Derive from AbstractVirtualSchemaAdapter
 local RlsAdapter = AbstractVirtualSchemaAdapter:new()
-local VERSION <const> = "1.1.0"
+local VERSION <const> = "1.2.0"
```

Comment:
If possible, add a test to verify consistent versions in rockspec and here, similiar to
https://github.com/exasol/exasol-driver-lua/blob/main/spec/unit/build_spec.lua

Reply:
https://github.com/exasol/row-level-security-lua/issues/121